src/app.js

```diff
@@ -1,8 +1,34 @@
 import config from './config';
 import Api from './Api';
+import InfoBarWidget from './InfoBarWidget';
+let components = $( 'body' ).data( 'wwt-components' );
 
-// eslint-disable-next-line no-unused-vars
-const a = new Api( { url: config.wikicolorUrl } ); // Test
-// TEST!
+// Initialize
+if ( !components ) {
+	// Store, so we don't append and create objects if they've been created
+	// TODO: We could potentially store all that data in some view model that's shared
+	// across the widgets.
+	components = {
+		widget: new InfoBarWidget(),
+		api: new Api( { url: config.wikicolorUrl } ),
+		$originalOutput: $( 'body' ).data( 'wwt-originalOutput' )
+	};
 
-OO.ui.alert( 'The extension is working! URL: ' + a.getAjaxURL( window.location.href ) );
+	$( 'body' ).data( 'wwt-components', components );
+
+	if ( $( 'body' ).hasClass( 'skin-vector' ) ) {
+		components.widget.$element
+			.insertBefore( $( '#content.mw-body' ) );
+	} else if ( $( 'body' ).hasClass( 'skin-monobook' ) ) {
+		$( '#content.mw-body' ).prepend( components.widget.$element );
```

Comment:
There's only one `#content` so the `.mw-body` is redundant (same with line 21)

Reply:
Good point, though I wanted to make sure I'm being contextually specific. I can remove it, or keep, either way works, it's more for context than function.

Reply:
It doesn't matter :) I think using only the ID will be faster (jQuery should use `getElementById`), but that has got to be by only microseconds.

Reply:
Changed.